templates/init_migration.sh.template

```diff
@@ -4,12 +4,12 @@
 cd "$(dirname "$0")"
 
 {{#DATABASE_GENERATED}}
-java -Ddw.database.user=postgres -Ddw.database.password="{{{ POSTGRES_DBPASSWORD }}}" -jar dockstore-webservice-*.jar db migrate web.yml --include 1.3.0.generated,1.3.1.consistency,1.4.0,1.5.0,1.6.0,1.7.0 | tee --append /dockstore_logs/webservice.out
+java -Ddw.database.user=postgres -Ddw.database.password="{{{ POSTGRES_DBPASSWORD }}}" -jar /home/dockstore-webservice-*.jar db migrate web.yml --include 1.3.0.generated,1.3.1.consistency,1.4.0,1.5.0,1.6.0,1.7.0 | tee --append /dockstore_logs/webservice.out
```

Comment:
Related to comments in #483, the `/home` directory should not contain anything except home directories for users on the system. This jar should be in `/home/<username>/.` (probably ubuntu, but not sure what kind of container this script is being run in).

Reply:
This is in the `/home` directory because the webservice Dockerfile put it there: https://github.com/dockstore/dockstore/blob/develop/Dockerfile#L19. The docker-compose file also puts things in the `/home` directory: https://github.com/dockstore/compose_setup/blob/develop/docker-compose.yml#L25. Should I be making changes to fix that?

Reply:
FWIW, the `home` directory thing is Linux best practice. That said, since this seems to be already in use in our various environments (and for a while), I would split this out the a new ticket to use `/home` properly and systematically (not 100% sure, it makes sense for Docker anyway)